plugins/kubernetes/app/models/kubernetes/api/pod.rb

```diff
@@ -3,7 +3,8 @@ module Kubernetes
   module Api
     class Pod
       INIT_CONTAINER_KEY = :'pod.beta.kubernetes.io/init-containers'
-      INGORED_AUTOSCALE_EVENT_REASONS = %w[FailedGetMetrics FailedRescale].freeze
+      INGORED_AUTOSCALE_EVENT_REASONS = %w[FailedGetMetrics FailedRescale, FailedGetResourceMetric,
+        FailedGetExternalMetric, FailedComputeMetricsReplicas].freeze
```

Comment:
%w and , don't work

... use `["Foo", "Bar"]`

Reply:
Also, let's split them one by line.

Reply:
already updated